src/scss/typography.scss

```diff
@@ -0,0 +1,37 @@
+.header2 {
+  font-size: $header-font-size;
+  text-transform: uppercase;
+  margin-bottom: $margin;
+  vertical-align: bottom;
+  text-align: center;
+  margin-bottom: 0;
+  i.fa-plus {
+    margin-left: $inner-padding;
+    cursor: pointer;
+  }
+}
+
+.header3 {
+  font-size: 15px;
+  text-transform: uppercase;
+  padding: 10px;
+  background-color: $mblue-bg;
+  color: #fff;
+}
+
+.header4 {
```

Comment:
I agree, I got stuck on what to name a thing after subheader, so I just went back to -#